.github/workflows/ci_workflows.yml

```
@@ -30,6 +30,9 @@ jobs:
           - os: ubuntu-latest
             python-version: 3.8
             toxenv: py38-test-sphinx40
```

If you like the other idea
```suggestion
            toxenv: py38-test-sphinx_oldest
```